packages/react-components/react-progress/stories/ProgressBar/index.stories.tsx

```diff
@@ -11,7 +11,7 @@ export { Shape } from './ProgressBarShape.stories';
 export { Thickness } from './ProgressBarThickness.stories';
 
 export default {
-  title: 'Preview Components/ProgressBar',
```

Comment:
You are missing removing this:
```
<!-- Don't allow prettier to collapse code block into single line -->
<!-- prettier-ignore -->
> **⚠️ Preview components are considered unstable:**
>
> ```jsx
>
> import { ProgressBar } from '@fluentui/react-components/unstable';
>
> ```
>
> - Features and APIs may change before final release
> - Please contact us if you intend to use this in your product
```
from ProgressBarDescription.md

Reply:
This has now been removed. Resolving this.